src/toolbox.ts

```diff
@@ -2536,37 +2552,37 @@ export class SubmitButtons extends ToolboxItem {
 
     add_event_listeners(): void {
         $(document).on("keypress.ulabel", (event) => {
-            const ctrl = event.ctrlKey || event.metaKey
-            if (ctrl &&
-                (
-                    event.key === "s" ||
-                    event.key === "S"
+            const ctrl = event.ctrlKey || event.metaKey;
+            if (ctrl
+                && (
+                    event.key === "s"
+                    || event.key === "S"
                 )
```

Comment:
here in particular it look kinda gross with the parentheses not aligned

Reply:
Yeah so this one was already half-unwrapped, so the rule just changed alignment. Wrapping it into one line doesn't get changed.

I will concede that for boolean operators having the linebreak after the operator does make this less likely to occur. I can make that change if you also prefer it.

Reply:
yeah changing it would be my vote

Reply:
Updated rule and applied in d1ad5ae